tests/bricks/test_wrappers.py

```diff
@@ -30,3 +31,47 @@ def test_as2d_ndim_leq_2():
     assert_allclose(
         f(numpy.ones(shape=(2, 3), dtype=theano.config.floatX)),
         3 * numpy.ones(shape=(2, 4), dtype=theano.config.floatX))
+
+
+def test_as2d_is_serializable():
+    brick = Linear(input_dim=3, output_dim=4, weights_init=Constant(1),
+                   biases_init=Constant(0))
+    wrapper = As2D(brick.apply)
+    wrapper.initialize()
+    cPickle.loads(cPickle.dumps(wrapper))
+
+
+def test_withaxesswapped_dim0_dim1_neq():
+    X = tensor.matrix('X')
+    brick = Linear(input_dim=2, output_dim=2, weights_init=Constant(1),
+                   biases_init=Constant(0))
+    wrapper = WithAxesSwapped(brick.apply, 0, 1)
+    wrapper.initialize()
+    brick.W.set_value(
+        numpy.asarray([[1, 2], [1, 1]], dtype=theano.config.floatX))
+    f = theano.function([X], wrapper.apply(X))
+    assert_allclose(f(numpy.arange(
+                        4, dtype=theano.config.floatX).reshape((2, 2))),
+                    numpy.array([[2, 4], [2, 5]]))
+
+
+def test_withaxesswapped_dim0_dim1_eq():
+    X = tensor.matrix('X')
+    brick = Linear(input_dim=2, output_dim=2, weights_init=Constant(1),
+                   biases_init=Constant(0))
+    wrapper = WithAxesSwapped(brick.apply, 0, 0)
+    wrapper.initialize()
+    brick.W.set_value(
+        numpy.asarray([[1, 2], [1, 1]], dtype=theano.config.floatX))
+    f = theano.function([X], wrapper.apply(X))
+    assert_allclose(f(numpy.arange(
+                        4, dtype=theano.config.floatX).reshape((2, 2))),
```

Comment:
This is apparently a syntax error. @bartvm , any idea why scrutinizer skipped this?